mobile/src/screens/marketplace.js

```diff
@@ -550,6 +551,13 @@ class MarketplaceScreen extends Component {
     updateExchangeRate(this.state.fiatCurrency[1], 'DAI')
   }
 
+  onWebViewNavigationStateChange = state => {
+    const urlMatch = state.url.match(/^https?:\/\/([^/?#]+)(?:[/?#]|$)/i)
```

Comment:
nit: We could probably use [URL Web API](https://developer.mozilla.org/en-US/docs/Web/API/URL).

Something like `new URL(state.url).hostname`.

Reply:
A lot cleaner your way thanks!